capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -198,6 +215,10 @@ class ClubPage extends Component {
 
               <div className='text-center'>
                 <div className='description'> {this.state.club.description} </div>
+                <div> Upcoming Meetings: </div>
```

Comment:
Does this have to be a div or can it be a span? Usually divs have more than 1 element inside them to categorize them

Reply:
I'll turn it into a span, and give it `display: block` as a style. Thanks for pointing this out!